addon/-private/class/modifier.ts

```diff
@@ -185,22 +224,34 @@ export default class ClassBasedModifier<S = DefaultSignature> {
       },
     });
 
+    if (DEBUG && !SEEN_CLASSES_FOR_DESTROYABLES.has(this.constructor)) {
```

Comment:
The deprecation just above needs to change from `false` to leverage `SEEN_CLASSES_FOR_DESTROYABLES.has(this.constructor)`, right now it always emits a deprecation

Reply:
Ahhh, thank you!